src/types.js

```diff
@@ -7,14 +9,32 @@ export type Webhook = {
   mydevices: boolean,
   productIdOrSlug: ?string,
   rejectUnauthorized: boolean,
-  requestType: string,
+  requestType: RequestType,
   responseTemplate: ?string,
   responseTopic: string,
   url: string,
 };
 
-export type WebhookRequestType = 'DELETE' | 'GET' |  'POST' | 'PUT';
+export type WebhookMutator = {
+  auth?: { Authorization: string },
+  deviceID?: boolean,
+  errorResponseTopic?: string,
+  event: string,
+  form?: { [key: string]: Object },
+  headers?:{ [key: string]: string },
+  json?: { [key: string]: Object },
```

Comment:
weird type actually. form, json, query - are different ways to pass all other webhook props to post request... our create() method in Webhook controller dosen't support those now.